styles/style.css

```
@@ -6,3 +6,55 @@ Green: #00B172
 White: #FFF
 
 */
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid black;
+}
+
+nav {
+    width: 350px;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+nav a {
+    text-decoration: none;
+    color: black;
+    font-family: 'Noto Sans KR', sans-serif;
+}
+
+.logo {
+    width: 17%;
+    margin-right: 1rem;
+    margin-bottom: 11px;
+}
+
+.sectionOne {
```

The ```width``` and ```height``` properties in this declaration can be omitted. This way the image stays full across the screen

```suggestion
.sectionOne {  background: url("../images/landing.jpg");
  background-size: cover;
  text-align: center;
  font-size: 40px;
  padding: 170px;
  font-family: "Noto Sans KR", sans-serif;
  color: azure;
  left: 0;
  }
```